src/Network/NWProtocolStack.cs

```diff
@@ -93,7 +93,7 @@ public void IterateProtocols (Action<NWProtocolOptions> callback)
 		public NWProtocolOptions InternetProtocol {
 			get {
 				var pHandle = nw_protocol_stack_copy_internet_protocol (GetCheckedHandle ());
-				return (pHandle == IntPtr.Zero) ? null : new NWProtocolOptions (pHandle, owns: true);
+				return (pHandle == IntPtr.Zero) ? null : new NWProtocolIPOptions (pHandle, owns: true);
```

Comment:
this will initialize `NWProtocolIPOptions` with the default (managed) values for all properties.
Those might not match the values of the native instance - and that would be very hard to debug/diagnose